examples/dependency/demo-with-conditions.yaml

```diff
@@ -43,14 +43,4 @@ spec:
         - valueFrom:
             dataOutputName: example-key
           toFieldPaths:
-            - "spec.key"
-      traits:
-        - trait:
-            apiVersion: serving.knative.dev/v1
-            kind: Route
-            metadata:
-              name: trait-test
-            spec:
-              traffic:
-                - configurationName: webapp
-                  percent: 100
```

Comment:
why deleting?

Reply:
It is lack of `serving.knative` support here,  and the example won't work if the user experiments the example. 

Or it is needed to add some knative support in prerequisite section, but i don't know if the `Route` trait can work correctly on this workload. 

So should i restore this section?

Reply:
I see, this section should be deleted.